src/psfmachine/tpf.py

```diff
@@ -502,6 +496,8 @@ def from_TPFs(
 
         if time_mask is not None:
             time_mask = np.copy(time_mask)[qual_mask]
+        # use or not the focus mask
+        focus_mask = focus_mask if do_focus_mask else None
```

Comment:
What will happen when `focus_mask` is `None`?

Reply:
it passes the None value to the class __init__ where is combined with the `time_mask`. If both none, then nothing is masked.